code/src/components/TaskCard.js/DeleteButton.js

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from "react";
+import { Tooltip, Dialog, DialogActions, DialogContent } from "@material-ui/core";
```

Comment:
When you are importing more than 2 items from one library it is better to organize them in a column to make them easier to read and debug.